python/kernmlops/kernmlops_benchmark/mongodb.py

```diff
@@ -0,0 +1,90 @@
+import subprocess
+from dataclasses import dataclass
+from typing import Literal, cast
+
+from data_schema import GraphEngine, demote
+from kernmlops_benchmark.benchmark import Benchmark, GenericBenchmarkConfig
+from kernmlops_benchmark.errors import (
+  BenchmarkNotInCollectionData,
+  BenchmarkNotRunningError,
+  BenchmarkRunningError,
+)
+from kernmlops_config import ConfigBase
+
+
+@dataclass(frozen=True)
+class BenchmarkConfig(ConfigBase):
+  
+  recordCount: int = 1000000
```

Comment:
please use lower snake case instead of camel case, i.e. `record_count`
note: Pascal case is fine for class names as you have done

Reply:
Also, after you make the above fix, please run `make defaults` to generate a new `defaults.yaml` and then commit those changes

Reply:
Also, it does not look like you use these configuration options at all anywhere
Please use them to configure the mongodb benchmark

Reply:
make defaults is consistently failing with this:
```
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/anish/KernMLOps/python/kernmlops/__main__.py", line 3, in <module>
    import cli
  File "/home/anish/KernMLOps/python/kernmlops/cli/__init__.py", line 7, in <module>
    import data_collection
  File "/home/anish/KernMLOps/python/kernmlops/data_collection/__init__.py", line 4, in <module>
    from data_collection import bpf_instrumentation as bpf
  File "/home/anish/KernMLOps/python/kernmlops/data_collection/bpf_instrumentation/__init__.py", line 5, in <module>
    from data_collection.bpf_instrumentation.bpf_hook import BPFProgram
  File "/home/anish/KernMLOps/python/kernmlops/data_collection/bpf_instrumentation/bpf_hook.py", line 4, in <module>
    from data_schema import CollectionTable
  File "/home/anish/KernMLOps/python/kernmlops/data_schema/__init__.py", line 7, in <module>
    from data_schema import perf
  File "/home/anish/KernMLOps/python/kernmlops/data_schema/perf/__init__.py", line 3, in <module>
    from data_schema.perf.perf_schema import (
  File "/home/anish/KernMLOps/python/kernmlops/data_schema/perf/perf_schema.py", line 3, in <module>
    from typing import Protocol, override
ImportError: cannot import name 'override' from 'typing' (/usr/lib/python3.10/typing.py)

```

Reply:
please run this inside docker, i.e.
```
make docker
make defaults
```